src/samples/ChildBenefitsClaim/CookiePageTable.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
Since the cookie page is a static page, you can add this table in cookie page. Anyhow this table is not reusable. or create a component folder 'cookiePage' and add both the components in there

Reply:
Thanks, resolved here [e312177](https://github.com/norm-l/odx/pull/123/commits/e312177a419463cdc4dbb3759f01a8bebda331e7)